unison-src/transcripts/cycle-update-4.md

```diff
@@ -0,0 +1,27 @@
+Not yet working: using update to establish a cycle.
+
+```ucm:hide
+.> builtins.merge
+```
+
+```unison
+ping : 'Nat
+ping _ = 1
+
+pong : 'Nat
+pong _ = !ping + 2
+```
+
+```ucm
+.> add
+```
+
+```unison
+ping : 'Nat
+ping _ = !pong + 1
+```
+
+```ucm
+.> update
+.> view ping pong
```

Comment:
Maybe add a comment here about what you expect to see when it _is_ eventually working 😄